src/main/resources/scripts/jigomerge-2.2.7.groovy

```diff
@@ -0,0 +1,596 @@
+/** Copyright Vincent Beretti vberetti|at|gmail<dot>com
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ * */
+public class SvnMergeTool {
```

Comment:
Is 2.2.7 any different from 2.2.6 ? If not, then we just need to change the .java file to accommodate this change.